ui/src/components/form/layout/layout.module.scss

```
@@ -16,15 +16,15 @@
 }
 
 .sectionTitle {
-  @include body-large();
+  @include body-large;
   color: var(--color-foreground);
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)